site/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "hacker-news-pwas",
+  "description": "Hacker News readers as Progressive Web Apps",
+  "scripts": {
+    "precache": "sw-precache --config=sw-precache-config.js --verbose"
```

Comment:
So here I think we can have a separate call that users can use for locally starting the site. For example:

```
  "scripts": {
    "precache": "sw-precache --config=sw-precache-config.js --root=_site --verbose",
    "start": "sw-precache --config=sw-precache-config.js && bundle exec jekyll serve"
  },
```
So npm run precache will make sure the service worker file gets generated in `_site` if done after `bundle exec jekyll build` and for a user to just run the site locally with the service worker they can just run `npm start`. Let me know if you agree and think this is a better approach.